src/stories/Button/button.css

```diff
@@ -0,0 +1,15 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+.test{
+  border: 5px solid black;
+}
+
+.test2{
+  border-radius: 30px;
+}
+
+.test3{
```

Comment:
This all seems to be a test CSS styling. Is is required to be merged in main branch?